Give examples of adding methods to interfaces as incompatible (#1503)
GoogleCloudPlatform/cloud-opensource-java: docs/JLBP-0003.md

    when a dependency is bumped to a higher major version.
  - Changing a method signature
  - Removing a method (deprecated or not)

>   - Adding a method to an interface unless you can provide a default implementation

Reviewer: ```suggestion
  - Adding a method to an interface without a default implementation
```

I don't think "can" is needed.
